### PR TITLE
Added a section for learning languages

### PR DESCRIPTION
**Pull request by spertch9a**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [forcoder.org](http://forcoder.org/) IT ebooks and e-Learning videos for download
 * [Packtpub](https://www.packtpub.com/) Free ebook available daily
 
+### Learn Languages
+* [Duolingo](https://www.duolingo.com/): The world's best way to learn a language!
+* [Learn a Language](https://www.learnalanguage.com/): Learn a language with hundreds of free lessons
+
```
Comment (Fcmam5) [inline README.md:183]: I suggest adding this at the bottom of the list, also update the summary

Comment (spertch9a) [reply README.md:183]: Done on my fork, but could you show me how I could modify my recent commit instead of creating a new one ?

Comment (Fcmam5) [reply README.md:183]: just push to your branch

Comment (spertch9a) [reply README.md]: Done

**Review by Fcmam5 — APPROVED** [review]
Awesome! Thank you Oussama

**Pull request merged**